### DISCUSSION TITLE
How to remove this light bulb icon thing?

### DISCUSSION THREAD

**Discussion by alimulap** [LazyVim/LazyVim#3953] [Q&A] [answered]
im not sure did i have this problem in the past, but this light bulb thing where if the cursor on some lines it will appear and not the other. 
I don't even know what's that for but its annoying and it messes with the screen layout.

Take a look at these

![image](https://github.com/LazyVim/LazyVim/assets/113952311/75e7b7a9-e642-4e1c-b178-f3457e2f4ff6)

![image](https://github.com/LazyVim/LazyVim/assets/113952311/eb539dcb-8373-4ba3-96f7-47b7518bbcbf)

I add the red circles to make it clear that the entire code shifted when there's a light bulb icon

It's also shifted when the current line number changed from one digit to two and vice versa

**Comment by folke** [comment, marked as answer]
That's not standard LAzyVim. You probbably added lspsaga?

**Reply by alimulap** [reply]
huh, you're right, that's why i thought i never saw this in the past.

i added lspsaga because i had a problem with lsp hover on vue project and someone told me to use lspsaga i thought it's just for hovering, turns out there are a lot going on that plugin.

Thanks!
